notary-commons/src/main/kotlin/sidechain/iroha/util/RequestHelper.kt

```diff
@@ -94,13 +92,31 @@ fun getAccountAsset(
         .map { queryResponse ->
             validateResponse(queryResponse, "account_assets_response")
 
-            val accountAsset = queryResponse.accountAssetsResponse.accountAssetsList
-                .find { it.assetId == assetId }
-
-            accountAsset?.balance ?: "0"
+            queryResponse.accountAssetsResponse.accountAssetsList
+                .map {
+                    it.assetId to it.balance
+                }.toMap()
         }
 }
 
+/**
+ * Get asset balance
+ *
+ * @param credential - iroha credential
+ * @param irohaNetwork - iroha network layer
+ * @param accountId - iroha account
+ *
+ * @return asset account balance if asset is found, otherwise "0"
+ */
+fun getAccountAsset(
```

Comment:
Maybe it would become clearer being `getAccountAssetBalance`

Reply:
done